commit 3f102ae926c01bccc9520a62cff833fde889ed6a
Author: Stephen Rothwell <sfr@canb.auug.org.au>
Date:   Sun Oct 9 10:35:16 2011 -0400

    mmc: using module_param requires the inclusion of moduleparam.h
    
    Commit "mmc: add module param to set fault injection attributes" adds
    a module_param to this file.  But it is relying on the old implicit
    "module.h is everywhere" behaviour, and without the explicit include
    of moduleparam.h, the pending module.h split up produces this error:
    
    core/debugfs.c:28:35: error: expected ')' before numeric constant
    
    Signed-off-by: Stephen Rothwell <sfr@canb.auug.org.au>
    Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
    Signed-off-by: Chris Ball <cjb@laptop.org>

diff --git a/drivers/mmc/core/debugfs.c b/drivers/mmc/core/debugfs.c
index da69aac4cc82..6045ea469362 100644
--- a/drivers/mmc/core/debugfs.c
+++ b/drivers/mmc/core/debugfs.c
@@ -7,6 +7,7 @@
  * it under the terms of the GNU General Public License version 2 as
  * published by the Free Software Foundation.
  */
+#include <linux/moduleparam.h>
 #include <linux/debugfs.h>
 #include <linux/fs.h>
 #include <linux/seq_file.h>